tests/static/js/formatters-internal/image.js

```diff
@@ -93,15 +71,15 @@ describe('image formatter', () => {
     });
 
     it('if can\'t get original ratio, use desired dimensions', () => {
-      const photoImgUrl = Formatters.image({url: photoUrl}, '1000x200', false).url;
+      const photoImgUrl = Formatters.image({url: photoUrl, width: 100}, '1000x200', false).url;
       expect(photoImgUrl).toEqual('https://dyn.mktgcdn.com/p/FOO/fit=contain,width=1000,height=200');
-      const photoWithPaddingImgUrl = Formatters.image({url: photoWithPaddingUrl}, '1000x200', false).url;
+      const photoWithPaddingImgUrl = Formatters.image({url: photoWithPaddingUrl, width: 0}, '1000x200', false).url;
       expect(photoWithPaddingImgUrl).toEqual('https://dyn.mktgcdn.com/p/FOO/1.0000/fit=contain,width=1000,height=200');
-      const oldFileImgUrl = Formatters.image({url: oldFileUrl}, '200x500', false).url;
+      const oldFileImgUrl = Formatters.image({url: oldFileUrl, height: 100}, '200x500', false).url;
```

Comment:
nit: can we update one of these to test both height and width being 0?

Reply:
done